app/uk/gov/hmrc/gform/auth/models/MaterialisedRetrievals.scala

```diff
@@ -96,6 +99,8 @@ case class VerifyId(id: String) extends AnyVal
 
 case class AnonymousRetrievals(sessionId: SessionId) extends MaterialisedRetrievals
 
+case class EmailRetrievals(sessionId: SessionId) extends MaterialisedRetrievals
```

Comment:
Why does this wraps `SessionId`? Why it is no user email?

Reply:
This has been changed, the case class now stores emailId